src/app/stacks/multivisionstack.cpp

```diff
@@ -48,7 +48,7 @@ string MultiVisionStack::getSettingsFileName() {
 
 void MultiVisionStack::createThreads(int number) {
   for (int i=0;i<number;i++) {
-    threads.push_back(new CaptureThread(i));
+    threads.push_back(new CaptureThread(i%4));
```

Comment:
Should this be `num_cameras` instead of `4`?

Reply:
Yeah, that should also work. Definitely looks better than the hard coded number :)